packages/rebber/__tests__/fixtures/inline-code.expected.tex

```diff
@@ -0,0 +1,9 @@
+a paragraph \texttt{with inline code}.
+
+\texttt{a
+multiline
+inlinecode}
+
+\texttt{a code with \textbackslash}
+
+\texttt{a \textbackslash{}text in \textbackslash{}LaTeX \textbackslash{}}
```

Comment:
Same here, at the end of the line: `\textbackslash{}\textbackslash{}` according to the markdown fixture (`\\`)